lib/src/presentation/scenes/splash/splash_screen_bloc.dart

```diff
@@ -1,21 +1,21 @@
 import 'package:rxdart/rxdart.dart';
-import 'package:xhabits/src/domain/splash_screen_use_case.dart';
+import 'package:xhabits/src/data/user_repository.dart';
 
 import 'splash_screen_state.dart';
 
 class SplashScreenBloc {
   BehaviorSubject<SplashScreenState> _splashStateSubject;
   Observable<SplashScreenState> get splashStateObservable => _splashStateSubject.stream;
   Future<dynamic> get closeStream => _splashStateSubject.close();
-  SplashScreenUseCase _useCase;
+  UserRepository _repository;
```

Comment:
@vpaniush why are you using class from data module on UI?

Reply:
@bogdanustyak fixed